torch/_export/passes/lift_constants_pass.py

```diff
@@ -199,7 +199,7 @@ def lift_constants_pass(
                 # Remove the parameterness of constant_val
                 if isinstance(constant_val, torch.nn.Parameter):
                     warnings.warn(
-                        f"{node.target} created when tracing {node.meta['stack_trace']} is a parameter. But"
+                        f"{node.target} created when tracing {node.meta.get('stack_trace', '<unknown stack>')} is a parameter. But"
```

Comment:
nit: this seems better than crashing for sure. But if you also found an example test where we found a param with no stacktrace, should we also try to fix that case (so we get a proper stacktrace?)

Reply:
this error was found in my previous version of the diff, and couldn't be reproduced with the same example I have with the current version... I think the reason is that in the buggy version, we produced some extra node in the graph which doesn't have the stacktrace. Nonetheless, I guess I will leave the code here as it's better to not crash with this warning.